README.md

```diff
@@ -1 +1,59 @@
-# catchpoint-demo-cinema-app
\ No newline at end of file
+# Catchpoint Demo Cinema App
+
+This is a demo cinema app a basic example of a cinema booking system. You can display a list of movies, view movie
+details, and book tickets for a movie.
+
+## Pre-requisites
+
+- Docker (https://docs.docker.com/get-docker/)
+- Docker Compose (https://docs.docker.com/compose/install/)
+- JQ (https://jqlang.github.io/jq/download/)
+- cURL (https://curl.se/download.html)
+
+## Setting up Catchpoint API Key
+
+1. Retrieve your api key from Catchpoint Portal
+2. Change `<API_KEY>` value from the
+    - For **Catchpoint** agent in the one of the following files:
```

Comment:
Seems that these files have been removed, we only have config-otel.properties files